venues/virtual/python/setup.py

```diff
@@ -866,21 +866,162 @@ def create_presentation_session(day, start, end):
+            'abstract': {'value': 'This is a place to leave your welcome message to the community!'}
+        }
+    }
+))
+
+welcome_wall_note = client.post_note(openreview.Note(invitation=welcome_wall_invitation_id,
+    readers=['everyone'],
+    writers=[conference_id],
+    signatures=[conference_id],
+    content={
+        'title': 'Welcome Wall Channel',
+        'abstract': 'This is a place to leave your welcome message to the community!'
+    }
+))
+
+comment_invitation_id = f"{virtual_group_id}/-/Comment"
+client.post_invitation(openreview.Invitation(
+    id=comment_invitation_id,
+    readers=['everyone'],
+    writers=[conference_id],
+    invitees=[conference_id, 'everyone'],
+    signatures=[conference_id],
+    reply={
+        'invitation': welcome_wall_invitation_id,
+        'readers': {'values': ['everyone']},
+        'writers': {'values-copied': [conference_id, '{signatures}']},
+        'signatures': {'values-regex': '~.*'},
+        'content': {
+            'comment': {
+                'order': 1,
+                'value-regex': '[\\S\\s]{1,5000}',
+                'description': 'Your comment or reply (max 5000 characters). Add formatting using Markdown and formulas using LaTeX. For more information see https://openreview.net/faq',
+                'required': True,
+                'markdown': True
+            }
+        }
+    }
+))
+
+## Create tag invitations
+## Bookmark presentations
+bookmark_invitation_id = f"{virtual_group_id}/-/Bookmark"
+client.post_invitation(openreview.Invitation(
+    id=bookmark_invitation_id,
+    readers=['everyone'],
+    writers=[conference_id],
+    invitees=['~'],
+    signatures=[conference_id],
+    reply={
+        'invitation': presentation_invitation_id,
+        'readers': {
+            'values-copied': [conference_id, '{signatures}']
+        },
+        'signatures': {
+            'values-regex': '~.*'
+        },
+        'content': {
+            'tag': {
+                'value': 'Bookmarked'
+            }
+        }
+    }
+))
+
+## Recommended presentations
+recommended_invitation_id = f"{virtual_group_id}/-/Recommendation"
+client.post_invitation(openreview.Invitation(
+    id=recommended_invitation_id,
+    readers=['everyone'],
+    writers=[conference_id],
+    invitees=[conference_id],
+    signatures=[conference_id],
+    reply={
+        'invitation': presentation_invitation_id,
+        'readers': {
+            'values-copied': [conference_id, '{signatures}']
+        },
+        'signatures': {
+            'values': [conference_id]
+        },
+        'content': {
+            'tag': {
+                'value-regex': r'[-+]?[0-9]*\.?[0-9]*'
+            }
+        }
+    }
+))
+
+## Free text tags
+tag_invitation_id = f"{virtual_group_id}/-/Tag"
+client.post_invitation(openreview.Invitation(
+    id=tag_invitation_id,
+    readers=['everyone'],
+    writers=[conference_id],
+    invitees=[conference_id],
```

Comment:
for free text tags, should invitees=['~']?

Reply:
yes!